### ISSUE TITLE
Query and display multiple LLM responses simultaneously - frontend

### ISSUE THREAD

**Issue by tubamos** [amosproj/amos2024ss06-health-ai-framework#204]
# Dependencies
- #210 
- #211 
- #174 
- #177 
- #188 

# Domain
app frontend

# Description
To fulfill the fundamental requirement of comparing responses coming from multiple LLMs, we can implement the display of parallel chat bubbles, one for each response from a different LLM.

The chat bubbles with responses that will be created will be as many as the number of LLMs that have been selected using the checkbox in the dropdown.

# Example
OpenAI has a similar UI in the web interface where there are two simultaneus responses produced that let the user select which is the best one. We could implement it in a similar way but for responses coming from different LLMs altogether.

![Image](https://github.com/amosproj/amos2024ss06-health-ai-framework/assets/166435676/35a8c74f-7146-4509-bd92-a086707d2932)

## Alternative:
Integrate multiple responses in the same chat response bubble.

# Resources
- This might be relevant to the horizontal scrolling
https://www.youtube.com/watch?v=cdP210MwnLw

- This might be relavant to the implementation in a single bubble:
https://www.cometchat.com/docs/ui-kit/react-native/interactive-card-bubble

# User Story
- As a user, 
- I want to select multiple LLMs and receive their responses in parallel chat bubbles, 
- so that I can analyze the differences in their answers.

# Acceptance Criteria
- [ ] The chat interface displays multiple responses from different LLMs in separate parallel chat bubbles.
- [ ] Each LLM response is presented in a distinct bubble, clearly indicating the originating LLM.
- [ ] The user can scroll horizontally to navigate between the chat bubbles that host the responses.

# Definition of Done
- [ ] The feature has been fully implemented.
- [ ] The feature has been manually tested and works as expected without critical bugs.
- [ ] The feature code is documented with clear explanations of its functionality and usage.
- [ ] The feature code has been reviewed and approved by at least one team member.
- [ ] The feature branches have been merged into the main branch and closed.
- [ ] The feature utility, function and usage have been documented in the respective project wiki on github.
